src/main/java/net/minecraftforge/fluids/Fluid.java

```diff
@@ -126,10 +124,15 @@
      */
     protected Block block = null;
 
+    //TODO Remove in 1.13
+    @Deprecated //Use constructor below
```

Comment:
I don't quite see the reason in keeping this constructor---code that uses it must be changed anyway (setRegistryName+register)...